packages/field/src/FormRegisteringMixin.js

```diff
@@ -24,16 +24,21 @@ export const FormRegisteringMixin = dedupeMixin(
         if (super.disconnectedCallback) {
           super.disconnectedCallback();
         }
+        formRegistrarManager.removeEventListener(
+          'all-forms-open-for-registration',
+          this._dispatchRegistration(),
+        );
         this._unregisterFormElement();
       }
 
       __setupRegistrationHook() {
         if (formRegistrarManager.ready) {
           this._dispatchRegistration();
         } else {
-          formRegistrarManager.addEventListener('all-forms-open-for-registration', () => {
-            this._dispatchRegistration();
-          });
+          formRegistrarManager.addEventListener(
+            'all-forms-open-for-registration',
+            this._dispatchRegistration(),
```

Comment:
Hey, just stumbled upon the PR and noticed that you invoke a function here rather than passing in a function. (I'm not an official reviewer of this PR)